frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailWrapper.unit.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ObjectDetailWrapper } from "metabase/visualizations/components/ObjectDetail/ObjectDetailWrapper";
+import { ObjectDetailProps } from "metabase/visualizations/components/ObjectDetail/types";
+import { testDataset } from "__support__/testDataset";
+
+function setup(options?: Partial<ObjectDetailProps>) {
+  render(
+    <ObjectDetailWrapper
+      data={testDataset as any}
+      question={
```

Comment:
Can we fix and set a proper question / table instance? Shouldn't be difficult, there are many tests that use `getMetadata` + `createMockEntitiesState`

Reply:
yep sounds good